packages/web/src/components/play-bar/desktop/PlayBar.js

```diff
@@ -503,22 +508,40 @@ class PlayBar extends Component {
 const makeMapStateToProps = () => {
   const getCurrentQueueItem = makeGetCurrent()
 
-  const mapStateToProps = (state, props) => ({
-    accountUser: getAccountUser(state),
-    currentQueueItem: getCurrentQueueItem(state),
-    playCounter: getCounter(state),
-    collectible: getCollectible(state),
-    isPlaying: getPlaying(state),
-    isBuffering: getBuffering(state),
-    playingUid: getPlayingUid(state),
-    playbackRate: getPlaybackRate(state),
-    lineupHasTracks: getLineupHasTracks(
+  const mapStateToProps = (state) => {
+    const premiumTrackSignatureMap = getPremiumTrackSignatureMap(state)
+    const lineupEntries = getLineupEntries(
       getLineupSelectorForRoute(state),
       state
-    ),
-    userId: getUserId(state),
-    theme: getTheme(state)
-  })
+    )
+    const lineupHasAccessibleTracks = lineupEntries.some((entry) => {
+      if (entry.kind !== Kind.TRACKS || !entry.id) return true
+
+      const { id } = entry
+      const {
+        is_premium: isPremium,
+        premium_content_signature: premiumContentSignature
+      } = getTrack(state, { id }) ?? {}
```

Comment:
get track could be empty if e.g. the track for the given id is not in the store, or if the id is null. in this case we should always be good, but added the {} fallback because of typescript and to be safe to not force the ! below

Reply:
Oh huh. If you had to add that for typescript, then I would think the type of `getTrack()` is wrong.